internal/pkg/db/db.go

```diff
@@ -93,14 +101,81 @@ func (backend *Backend) AddDocument(document *sbom.Document) error {
 func (backend *Backend) GetDocumentByID(id string) (*sbom.Document, error) {
 	document, err := backend.Retrieve(id, nil)
 	if err != nil {
-		backend.Logger.Warn("Document could not be retrieved", "id", id, "err", err)
+		backend.Logger.Debug("Document could not be retrieved", "id", id, "err", err)
 
 		return nil, fmt.Errorf("failed to retrieve document: %w", err)
 	}
 
 	return document, nil
 }
 
+func (backend *Backend) GetDocumentByIDOrAlias(id string) (*sbom.Document, error) {
+	document, idErr := backend.GetDocumentByID(id)
```

Comment:
This still needs to check the error, which should only be returned if an actual database error occurred. So just rename to `err`

Reply:
For now I'm logging the error but I'll raise an issue in `protobom-storage` because `GetDocumentByID` is also returning an error if a document with a specific ID is not found.

Reply:
There was no need for an issue. I just swapped usage of `backend.Retrieve` with `backend.GetDocumentsByID` in `backend.GetDocumentByID`.